src/topology/alexandroff.lean

```diff
@@ -45,6 +45,10 @@ In this section we define `alexandroff X` to be the disjoint union of `X` and `
 /-- The Alexandroff extension of an arbitrary topological space `X` -/
 def alexandroff (X : Type*) := option X
 
+-- Will only work if locale `alexandroff` is open
```

Comment:
```suggestion
/-- The repr uses the notation from the `alexandroff` locale -/
```
It works just fine without the locale open, you just can't use the result as an expression.

Reply:
Yeah, that's what I meant.